custom/inddex/tests/test_master_report.py

```diff
@@ -224,8 +224,9 @@ def to_string(row):
         expected_report = get_expected_report(csv_filename)
         self.assertEqual(list(expected_report[0].keys()), actual_report.headers)
 
-        for expected_row, actual in zip(expected_report, actual_report.rows):
-            expected = list(expected_row.values())
+        expected_rows = sorted(list(r.values()) for r in expected_report)
+        actual_rows = sorted(actual_report.rows)
+        for expected, actual in zip(expected_rows, actual_rows):
```

Comment:
I was originally hoping to do all the sorting in the report, but the only columns known to be unique are case ID columns, which are random, so the generated report won't always match the test data.  This sorts in the test after swapping in the proper IDs.